File: RepoTransBench/target_projects/Python/C++/stjordanis_betterprompt/public_tests/test_public_init_module.cpp
#include <gtest/gtest.h>
#include "betterprompt.h"

TEST(TestPublicInitModule, AllExportsViaDict) {
    for (const auto& name : betterprompt::get_all_exports()) {
        // Cannot check __dict__ in C++ – just that the symbol exists in __all__
        SUCCEED();
    }
}

File: RepoTransBench/target_projects/Python/Rust/stjordanis_betterprompt/public_tests/test_public_init_module.rs
use betterprompt::meta::__all__;

#[test]
fn test_public_all_exports() {
    for &name in __all__ {
        // Just check it's referenced in meta (simulate public __dict__ access)
        match name {
            "get_from_dict_or_env" | "calculate_perplexity" | "call_openai" | "DummyOpenAICompletion" | "openai" => { /* exists */ }
            _ => panic!("Unexpected export: {}", name),
        }
    }
}